docs/changelog.rst

```diff
@@ -5,36 +5,42 @@ Changelog
 
 We use a one-line description of every pull request.
 
-.. to obtain the merged PRs since a specific release, e.g. v0.2.0 use
-.. `$ git log --merges --first-parent master --oneline  master...v0.2.0`
-.. to obtain the contributor, use
-.. `$ git shortlog -sne --no-merges master...v0.2.0
-
-.. RELEASE TEMPLATE
-..
-.. `X.Y.Z < github link >`__ (Month Day, YEAR)
-.. -------------------------------------------
-..
-.. Summary
-.. +++++++
-..
-.. - Released Month Day, YEAR
-.. - N contributors
-..
-.. **Description**
-..
-.. . . .
-..
-.. **Contributors:**
-..
-.. In alphabetical order by last name:
-..
-.. - . . .
-..
-.. Pull Requests
-.. +++++++++++++
-..
-.. - [#XXX] TITLE (AUTHOR)
```

Comment:
Instead of simply removing it, can we save this information  under a new section for developers/maintainers under `/docs/contribute.rst`?

Reply:
I found it easier just to copy an older release and adapt the info....